tests/server.ts

```diff
@@ -94,11 +96,57 @@ describe('webhooks', () => {
         .send({})
       expect(response.status).not.toBe(401)
     })
+
     it('should return 403 without auth header', async () => {
       const response = await request(app)
         .post('/api/webhooks/github-release')
         .send({})
       expect(response.status).toBe(403)
     })
+
+    it('bad url: should return 400', async () => {
```

Comment:
can these test cases be more descriptive in regards to setting the context?

"should return 400 if webhook URL is bad"
"should return 204 if event action is not 'released'"